include/deal.II/grid/grid_generator.h

```diff
@@ -501,7 +507,11 @@ namespace GridGenerator
    * point will be the sum of these two vectors.  Colorizing is done in the
    * same way as in hyper_rectangle().
    *
-   * @note This function is implemented in 2d only.
+   * @note This function is implemented in 2d only. 
+   * 
+   * @param tria Triangulation to create.
+   * @param corners Adjacent points. 
+   * @param colorize Assign different boundary ids if set to true.
```

Comment:
Would you mind please adding a reference to the glossary entry on colorization, similar to that which is done for the other functions?